perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -582,6 +582,7 @@ public Application call(ApiCaller ac, Deserializer parms) throws PerunException
 	 * Creates a new application.
 	 * The method triggers approval for VOs with auto-approved applications.
 	 *
+	 * @deprecated
```

Comment:
Let docs reader know about the new way: ``@deprecated See submitApplication()``

Reply:
Done